Example/SettingsTableViewController.swift

```diff
@@ -14,8 +14,9 @@ class SettingsTableViewController: UITableViewController {
         addTapGesture()
     }
 
-    override func viewDidAppear(_ animated: Bool) {
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
as a best practice don't forget to call the super when overriding

Reply:
Done